### ISSUE TITLE
MQTT login

### ISSUE THREAD

**Issue by zamzon** [jalla2000/kamstrup-mqtt#2]
Hi

Where do I insert my broker username and password in the sketch.ino?

Is it here?:
if (client.connect(clientId.c_str(), "mqttclient", "YourPassword")) {
 
mqttclient= username
YourPassword = password

**Comment by jalla2000** [comment]
Yes, that is exactly right. It should have been two variables next to the wifi settings instead.
